doc/man/8/mount.ceph.rst

```diff
@@ -61,10 +61,13 @@ Basic
     for autodiscovery of monitor addresses and auth secrets. The default is
     to use the standard search path for ceph.conf files.
 
-:command:`mds_namespace=<fs-name>`
+:command: `fs=<fs-name>`
     Specify the non-default file system to be mounted. Not passing this
     option mounts the default file system.
 
+:command: `mds_namespace=<fs-name>`
+    A synonym of "fs=" and its use is deprecated.
```

Comment:
How about this instead?

A synonym of "fs=" (Deprecated)

Reply:
Not sure, the first impression of it seems saying the "fs=" is deprecated.